k8s/namespaces/money-claims/cmc-legal-frontend/cmc-legal-frontend.yaml

```diff
@@ -8,16 +8,8 @@ spec:
   rollback:
     enable: true
     retry: true
-  chart:
```

Comment:
Can you look at how you're doing this? Here you're removing the chart from the base - and then it gets repeated in all the environment patches. Patches should just be for anything you want to overwrite/env specific additions

Reply:
Typically this would be the case. 

However some of the environments have a different repository source. This would result in a chart section with two different sources if the chart section is also in the base chart. e.g.

```
spec:
  chart:
    git: git@github.com:hmcts/hmcts-charts
    name: cmc-legal-frontend
    path: stable/cmc-legal-frontend
    ref: master
    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
    version: 1.2.0
```

Reply:
Resolved (ithc and perftest had some old pinned versions, now all using ref: master)